File: app/src/main/java/knf/kuma/tv/details/ChaptersListRow.java
package knf.kuma.tv.details;

import androidx.leanback.widget.HeaderItem;
import androidx.leanback.widget.ListRow;
import androidx.leanback.widget.ObjectAdapter;

public class ChaptersListRow extends ListRow {
    public ChaptersListRow(HeaderItem header, ObjectAdapter adapter) {
        super(header, adapter);
    }

    public ChaptersListRow(long id, HeaderItem header, ObjectAdapter adapter) {
        super(id, header, adapter);
    }

    public ChaptersListRow(ObjectAdapter adapter) {
        super(adapter);
    }
}

File: app/src/main/java/knf/kuma/tv/details/ChaptersListRow.kt
package knf.kuma.tv.details

import androidx.leanback.widget.HeaderItem
import androidx.leanback.widget.ListRow
import androidx.leanback.widget.ObjectAdapter

class ChaptersListRow : ListRow {
    constructor(header: HeaderItem, adapter: ObjectAdapter) : super(header, adapter)

    constructor(id: Long, header: HeaderItem, adapter: ObjectAdapter) : super(id, header, adapter)

    constructor(adapter: ObjectAdapter) : super(adapter)
}
